src/magento/magento-account-secure.md

```diff
@@ -43,7 +43,7 @@ In order to use two-factor authentication, you must have a two-factor authentica
    You can either scan the QR code using the two-factor authentication application or manually enter the code into your two-factor authentication application. This will sync your two-factor authentication application with your Magento.com account and allow your two-factor authentication application to generate verification codes that Magento.com will accept.
```

Comment:
```suggestion
   You can either scan the QR code using the TFA application or manually enter the code into your TFA application. This will sync your TFA application with your Magento.com account and allow your TFA application to generate verification codes that Magento.com will accept.
```